client/src/components/Cards/cards.tsx

```diff
@@ -26,21 +28,117 @@ export const Cards = () => {
 				<>
```

Comment:
I'm facing an issue with the Link element from Next, once the modal/dialog is open, and hit ESC the URL is not updating. I need to look into this.